docs/src/maintainer-guide/governance.md

```diff
@@ -26,7 +26,30 @@ Contributors are community members who contribute in concrete ways to the projec
 
 Contributors have read-only access to source code and so submit changes via pull requests. Contributor pull requests have their contribution reviewed and merged by a TSC member. TSC members and Committers work with Contributors to review their code and prepare it for merging.
 
-As Contributors gain experience and familiarity with the project, their profile within, and commitment to, the community will increase. At some stage, they may find themselves being nominated for committership by an existing Committer.
+As Contributors gain experience and familiarity with the project, their profile within, and commitment to, the community will increase. At some stage, they may find themselves being nominated as either a Website Team Member or Committer by an existing Website Team Member or Committer.
+
+### Website Team Member
+
+Website Team Members are community members who have shown that they are committed to the continued maintenance of eslint.org through ongoing engagement with the community. Website Team Members are given push access to the `eslint.org` GitHub repository and must abide by the project's [Contribution Guidelines](../developer-guide/contributing/).
```

Comment:
> Website Team Members are given push access to the `eslint.org`

I wonder if there would be no access to `playground` it is part of docs as well and `eslint/eslint` as docs folder falls under that repo?

Reply:
We are planning to move playground to the eslint.org repo. I will probably create a PR for this soon.

Reply:
Because eslint/eslint also contains the main code for the project, I think we won’t include write access, at least initially. We can always revisit. Following the principle of least privilege.